packages/core/test/combinator/sample-test.js

```diff
@@ -30,15 +30,15 @@ describe('sample', () => {
     const s = sample(Array, makeEvents(1, n), never())
 
     return collectEvents(s, ticks(n))
-      .then(events => is(0, events.length))
+      .then(events => eq(0, events.length))
```

Comment:
yes, yes that would be interesting... i love pushing the assert libraries and such to FP land 😍